tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "rootDir": "./src",
     "strict": true,
     "noImplicitAny": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "sourceMap": true,
+    "inlineSourceMap": false
```

Comment:
Setting 'inlineSourceMap' to false is appropriate if you want to keep the source maps separate from the compiled files. This can help reduce the size of the output files and is a common practice in production environments.